src/modules/search/components/SearchBar/SearchBar.test.js

```diff
@@ -21,17 +20,19 @@ jest.mock('src/modules/search/hooks/useSearch', () => ({
 jest.mock('@token/fungible/hooks/queries');
 
 describe('SearchBar', () => {
-  const props = {
-    history: {
+  const config = {
+    renderType: 'mount',
+    historyInfo: {
```

Comment:
Why rename this to historyInfo, it is supposed to be the history?

Reply:
@oskarleonard `historyInfo` is the name of the property used to pass history into the smartRender test helper config. I used the name so it doesn't clash with the history variable that exists in the same test helper file

Reply:
But wont it make more sense to call it just history if that is what it is supposed to represent. historyInfo feels more confusing to me, like it is a variant of the history object. Since it is scoped in `config` object, `history` wont be that confusing? 🤔 Or if we want to really have a different name, maybe `historyObject` or `historyFields`?

This snippet will probably be copy pasted a lot since it is the only place using the new `smartRender`. Which is why i am giving more thought to the naming.

Reply:
The test helper where historyInfo is used already has a history variable. `historyObject` is an option and I considered it but it didn't feel quite right. I also wanted to make it consistent with the other keys for the config object.

Reply:
Ok, than lets go for as it is